paths/entities/items/sitelinks/list.json

```diff
@@ -12,5 +13,18 @@
             "404": { "$ref": "../../../../responses.json#/NotFound" },
             "default": { "$ref": "../../../../responses.json#/UnexpectedError" }
         }
+    },
+    "post": {
+        "tags": [ "items" ],
+        "summary": "Retrieves an item's sitelinks",
```

Comment:
Probably the description should say something like "Adds item's sitelinks"

Reply:
Very true :+1: Thanks :smile: